packages/react-composites/src/composites/MeetingComposite/adapter/MeetingAdapter.ts

```diff
@@ -93,7 +90,11 @@ export interface MeetingAdapter
       ChatAdapterHandlers,
       'fetchInitialData' | 'sendMessage' | 'sendReadReceipt' | 'sendTypingIndicator' | 'loadPreviousChatMessages'
     >,
-    MeetingAdapterSubscriptions {}
+    MeetingAdapterSubscriptions {
+  joinMeeting(microphoneOn?: boolean): void;
+  leaveMeeting(forEveryone?: boolean): Promise<void>;
```

Comment:
remove `forEveryone`, this can work for calls but you can't make everyone leave a chat thread

Reply:
`leaveMeeting(forEveryone)` anyway sounds weird to me. You can remove others, you can't really leave for them :)

If we want the functionality to kick everyone, we can add a separate method removeAllParticipants(), although I'm not sure why we'd want that functionality at all.

Reply:
It was purely reflecting the call sdk that has leaveCall(forEveryone) - but yeah removing the forEveryone and just having `leaveMeeting`